src/routes/AgreementViewRoute/AgreementViewRoute.js

```diff
@@ -214,6 +214,7 @@ class AgreementViewRoute extends React.Component {
     }
   }
 
+  /* istanbul ignore next */
```

Comment:
Any reason you are ignoring all the callbacks here for code coverage? All of these callbacks can be covered via the tests. Please double check.